frontend/app_flowy/packages/appflowy_editor/lib/src/render/rich_text/embed_code.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:appflowy_editor/src/document/node.dart';
+import 'package:appflowy_editor/src/editor_state.dart';
+import 'package:appflowy_editor/src/infra/flowy_svg.dart';
+import 'package:appflowy_editor/src/render/rich_text/default_selectable.dart';
+import 'package:appflowy_editor/src/render/rich_text/flowy_rich_text.dart';
+import 'package:appflowy_editor/src/render/rich_text/rich_text_style.dart';
+import 'package:appflowy_editor/src/render/selection/selectable.dart';
+import 'package:appflowy_editor/src/service/render_plugin_service.dart';
+import 'package:flutter/material.dart';
+
+class EmbedCodeTextNodeWidgetBuilder extends NodeWidgetBuilder<TextNode> {
```

Comment:
I don't think we need to create a new node widget to process the embed code. The embed code style should be like the bold, italic style. We should integrate this feature into rich_text.dart.